server/views/pages/analytics/protectedCharacteristicTemplate.njk

```diff
@@ -28,7 +33,7 @@
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <form id="form-select-characteristic" method="get" action="/analytics/protected-characteristic" novalidate>
+      <form id="form-select-characteristic" method="get" action={{ selectProtectedCharacteristicLink }} novalidate>
```

Comment:
you can even drop the action attribute as we POST the current url

Reply:
Good spot! When I originally added that I made the comment https://github.com/ministryofjustice/hmpps-incentives-ui/pull/192#discussion_r880478314 but just re-tested and we don't need it 😄

Reply:
i'd (of course) forgotten why it was there